R/longData.R

```diff
@@ -51,9 +51,9 @@ longDataConstructor <- R6::R6Class(
         strata = NULL,
 
 
-        #' @field visit_ice A list indexed by subject storing the visit level which
+        #' @field ice_visit_index A list indexed by subject storing the visit index number which
```

Comment:
```suggestion
        #' @field ice_visit_index A list indexed by subject storing the index number of the first visit affected by the ICE. It is equal to the number of visits plus 1 in case of no ICE.
```